### ISSUE TITLE
Issue Installing on HPC

### ISSUE THREAD

**Issue by roshankern** [sunbeam-labs/sunbeam#403]
I am struggling to install sunbeam on my university's high performance clusters. After a couple attempts to install sunbeam with my existing conda/mamba install, I removed both and freshly reinstalled conda and mamba. I followed the [quick start guide](https://sunbeam.readthedocs.io/en/stable/quickstart.html#quickstart) with `-v` to show verbose outputs and tried installing with and without the `--no_mamba` argument. In each case, the `/.install.sh` script had the following output:
<img width="1127" alt="image" src="https://github.com/sunbeam-labs/sunbeam/assets/105665675/63ce7fab-4803-4465-8261-b08cb794c984">
<img width="1127" alt="image" src="https://github.com/sunbeam-labs/sunbeam/assets/105665675/fed53c15-7803-4e69-b8c9-d42d8e4f28a7">
<img width="1129" alt="image" src="https://github.com/sunbeam-labs/sunbeam/assets/105665675/3ba2fe8e-2e92-4ef5-b757-3cd4de4100cc">

It seems like the main issue is `C compiler cannot create executables`, but I have confirmed that I am able to use `GCC` and `CC` on the HPC. I would like to check out the `config.log` file to see what issues the C compiler is having but I was unable to locate this file. Is someone able to help me diagnose the issue? 

Thanks in advance!

**Comment by Ulthran** [comment]
Hi @roshankern thanks for using sunbeam! I can't say I've ever seen this error before but it looks like something internal to conda and wheel that's going wrong. What's the architecture of the machine you're running this on?

One thing you might try to get around this issue is creating the env from a pin file with `conda create --name sunbeam3 --file stable_env/v3.0.0.txt`.

As another alternative, I just released v4.0.0-rc.3, which is still a pre-release but you could try that out. The actual v4 release should be coming soon (fingers crossed).

 - Charlie

**Comment by roshankern** [comment]
Hey @Ulthran, thanks for your help thus far! Below is the output of `lscpu` when interacting with the HPC. It looks like the architecture of the machine is `x86_64`
![image](https://github.com/sunbeam-labs/sunbeam/assets/105665675/efab366e-7ab0-450b-a612-8ace58d73a99)

As for creating the environment from a pin file, the command you suggested first failed with `CondaHTTPError: HTTP 404 NOT FOUND for url <https://conda.anaconda.org/conda-forge/linux-64/gdk-pixbuf-2.36.12-haf2c3b9_1004.tar.bz2>`. It seems like this package is no longer available from the given link. After removing this line from the `stable_env/v3.0.0.txt` file, the environment was successfully created. However, with this method `sunbeam` is not added to PATH so I proceeded with the next suggestion.

I was able to install `v4.0.0-rc.3` and all tests passed. Thanks for your help!
